### PR TITLE
Feat views

### PR DESCRIPTION
**Pull request by justend29**
## Background

PR to start creating a baseline that we can work off of.
Contains basic views and some basic auth flow.

## Changes

A lot. Honestly, we're just getting a starting ground going. 
Just merged the basic auth views into this branch. Will continue actually performing auth everywhere, but the views seem be cooperating now.

## Review

How much did I break?

## TODO

literally everything. Plenty of issues with overfull boxes (especially when the keyboard pops up), so we'll need to fix those views to render properly.

### CODE REVIEW

File: amplify/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Getting Started with Amplify CLI
+This directory was generated by [Amplify CLI](https://docs.amplify.aws/cli).
+
+Helpful resources:
+- Amplify documentation: https://docs.amplify.aws
+- Amplify CLI documentation: https://docs.amplify.aws/cli
+- More details on this folder & generated files: https://docs.amplify.aws/cli/reference/files
+- Join Amplify's community: https://amplify.aws/community/
```
Comment (StephenCole19) [inline amplify/README.md:8]: Do we need this file?

File: lib/views/homepage.dart [hunk 2]
```diff
@@ -1,54 +1,152 @@
...
               decoration: const InputDecoration(
                 icon: Icon(Icons.confirmation_number),
                 hintText: 'What license place would you like to examine?',
                 labelText: 'License Plate Number *',
               ),
               onSaved: (String? value) {
-                // This optional block of code can be used to run
-                // code when the user saves the form.
+                print("SAAAAAAAAAAAVING");
+                enableExamination = value != null && value.isNotEmpty;
```
Comment (StephenCole19) [inline lib/views/homepage.dart:76]: lmao

File: lib/views/manageofficerspage.dart [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack")
```
Comment (StephenCole19) [inline lib/views/manageofficerspage.dart:33]: :eyes:

File: amplify/hooks/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Command Hooks
+
+Command hooks can be used to run custom scripts upon Amplify CLI lifecycle events like pre-push, post-add-function, etc.
+
+To get started, add your script files based on the expected naming convention in this directory.
+
+Learn more about the script file naming convention, hook parameters, third party dependencies, and advanced configurations at https://docs.amplify.aws/cli/usage/command-hooks
```
Comment (StephenCole19) [inline amplify/hooks/README.md:7]: Do we need this file?

**Review by StephenCole19 — APPROVED** [review]
Looks good to me! Maybe remove the readme for amplify unless you want it and it would probably be a good idea to remove the "Ballsack" consts incase bil or the TA review but that's up to you. 👍

File: lib/views/manageofficerspage.dart (hunk 3 above)
Comment (justend29) [reply lib/views/manageofficerspage.dart:33]: @StephenCole19 these are definitely valid parking officers from the DB

Comment (StephenCole19) [reply lib/views/manageofficerspage.dart:33]: @justend29 I believe you, continue on 🥇

**Pull request merged**